src/store/actions/searching-project-user/project/project.js

```diff
@@ -93,23 +93,29 @@ export const _onSubmitFormAnswers = (answers, organizationId, projectId) => {
       uid: studentUID,
     } = student;
 
+    console.log("getState",getState)
     console.log('submitting form answers')
+    console.log("uid",student.uid)
+    console.log("studentId",studentUID)
+    console.log("studentEmail",student.studentEmail)
+    console.log("orgId",orgId)
     try {
       await firebase_db
         .collection("organization")
-        .doc(`${organizationId}`)
+        .doc(`${orgId}`)
         .collection("projects")
         .doc(`${projectId}`)
         .collection("formSubmission")
-        .doc(`${studentUID}`)
+        // .doc(`${studentUID}`)
```

Comment:
Why did you delete this line? It is required to push the data to the exact student @TrNgTien